features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx

```diff
@@ -94,20 +115,47 @@ const useStyles = makeStyles((theme) => ({
 function UploadTheme() {
     const classes = useStyles();
     const [themeFile, setThemeFile] = useState([]);
+    const [fileName, setFileName] = useState('');
```

Comment:
Do we need to initialize the state with an empty string ?

Reply:
Fixed with `2d41afa`